tools/pythonpkg/src/pyrelation.cpp

```diff
@@ -94,8 +94,6 @@ void DuckDBPyRelation::Initialize(py::handle &m) {
 	         py::arg("aggregation_columns"))
 	    .def("describe", &DuckDBPyRelation::Describe,
 	         "Gives basic statistics (e.g., min,max) and if null exists for each column of the relation.")
-	    .def("union", &DuckDBPyRelation::Union,
```

Comment:
why is this removed?

Reply:
As mentioned in the commit, it's a duplicate